main.py

```diff
@@ -26,7 +26,7 @@ class Pokemon(BaseModel):
     legendary: bool
```

Comment:
It's recommended to create input and output Schemas for each route based on what we read and send.
something like

input schemas:
PokemonPostPutInputSchema (most of the fields will be mandatory in input)
PokemonPatchInputSchema (all the fields should be optional, as the user may change only a few columns)

output schemas:
PokemonGetOutputSchema (what columns do you want to send for get route ??)
PokemonGetAllOutputSchema (this is mostly a list of the above Pokemon object)
PokemonPostPatchPutOutputSchema (for POST/PUT/PATCH we can send the same set of columns as output)